packages/kit/src/core/build/index.js

```diff
@@ -392,6 +385,15 @@ async function build_server(
 			const get_hooks = hooks => ({
 				getSession: hooks.getSession || (() => ({})),
 				handle: hooks.handle || (({ request, resolve }) => resolve(request)),
+				handleError:
+					hooks.handleError ||
+					(({error}) => {
+						if (error.frame) {
+							console.error(error.frame);
+						}
+						console.error(error.stack);
+						error.stack = options.get_stack(error);
```

Comment:
We should probably do this even if the user provides a `handleError` method

Reply:
I'm not really even sure what this line is doing, since `error` isn't returned anywhere. I'm not super familiar with JS - does this modify the reference to `error`, where the caller's `error` object will now have the `error.stack` defined here?

I pasted this from the old `handle_error()` to preserve however that was working before, so I don't know what this is doing here.

Reply:
> does this modify the reference to error, where the caller's error object will now have the error.stack defined here?

yep!

Reply:
Cool, I will look into adding that everywhere.

Reply:
I don't actually think this is a good idea.

`options.get_stack(error)` is just doing `String(error)`, which I'm seeing actually drops the stack trace that I'm currently receiving.

```js
// before
  err: {
    name: 'FetchError',
    message: 'request to http://doesnotexist.foo/ failed, reason: getaddrinfo ENOTFOUND doesnotexist.foo',
    code: 'ENOTFOUND',
    stack: 'FetchError: request to http://doesnotexist.foo/ failed, reason: getaddrinfo ENOTFOUND doesnotexist.foo\n' +
      '    at ClientRequest.<anonymous> (file:///home/andrew/Work/src/github.com/andrew-womeldorf/kit/packages/kit/dist/install-fetch.js:1510:11)\n' +
      '    at ClientRequest.emit (events.js:315:20)\n' +
      '    at ClientRequest.EventEmitter.emit (domain.js:467:12)\n' +
      '    at Socket.socketErrorListener (_http_client.js:469:9)\n' +
      '    at Socket.emit (events.js:315:20)\n' +
      '    at Socket.EventEmitter.emit (domain.js:467:12)\n' +
      '    at emitErrorNT (internal/streams/destroy.js:106:8)\n' +
      '    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n' +
      '    at processTicksAndRejections (internal/process/task_queues.js:80:21)'
  }
```

```js
// after
  err: {
    name: 'FetchError',
    message: 'request to http://doesnotexist.foo/ failed, reason: getaddrinfo ENOTFOUND doesnotexist.foo',
    code: 'ENOTFOUND',
    stack: 'FetchError: request to http://doesnotexist.foo/ failed, reason: getaddrinfo ENOTFOUND doesnotexist.foo'
  }
```

Reply:
That's in `build`. In `dev` it does something much different. Also, the build code says it does that for security, so I'm not sure we'd just want to drop it. I don't think `new String` should drop the successive lines

Reply:
Okay, I've added a solution that I think should be fine. Let me know if this is what you had in mind